app/my_jss/index.js

```diff
@@ -0,0 +1,180 @@
+
+
+function loginFunction(){
+  window.location.href = "login.html";
+}
+
+var dataInformation = JSON.parse(playlistsJSON);
+var allPlaylist = document.getElementsByClassName('section-absolute')[0];
+var popUpPlaylist = document.getElementById('playlistAbsolut');
+
+
+
+generatePlaylists();
+
+
+function generatePlaylists(){
+
+for (var i=0; i < dataInformation.length; i++)
+  {
+      createElement(dataInformation[i]);
+  }
+}
+function createElement(elem){
+  var bigDiv = document.createElement('div');
+  bigDiv.setAttribute("class","individual-playlist");
+  allPlaylist.appendChild(bigDiv);
+  var middleDiv= document.createElement('div');
+  middleDiv.setAttribute("id","individual-playlist-button");
+  //ADD HERE THE EVENT
+  bigDiv.appendChild(middleDiv);
+
+  middleDiv.addEventListener('click', function(){
+    var title = elem.title;
+    showPlaylist(title);
+  });
+  /////
+  var imgBut = document.createElement('img');
+  imgBut.setAttribute("src", "../core/assets/play-icon.png");
+  imgBut.naturalHeight;
+  imgBut.naturalWidth;
+  middleDiv.appendChild(imgBut);
+  var paraGraph = document.createElement('p');
+  paraGraph.innerHTML = 'Explore Playlist';
+  middleDiv.appendChild(paraGraph);
+  /////
+  var imgObj = document.createElement('img');
+  imgObj.setAttribute("src",elem.imageLarge);
+  imgObj.naturalHeight;
+  imgObj.naturalWidth;
+  bigDiv.appendChild(imgObj);
+  //////
+  var hObj = document.createElement('h3');
+  hObj.innerHTML = elem.title;
+  bigDiv.appendChild(hObj);
+  ////
+  var pObj = document.createElement('p');
+  pObj.innerHTML = elem.description;
+  bigDiv.appendChild(pObj);
+
+}
+
+function showPlaylist(title_string){
+
+  for(var i=0; i<   dataInformation.length;i++)
+  {
+    if(dataInformation[i].title == title_string)
+    {
+      constructIndividualPlaylist(dataInformation[i]);
+    }
+  }
+  popUpPlaylist.style.display = 'inline';
+}
+
+function constructIndividualPlaylist(elem){
+
+ removeChildsIfExits();
+//popUpPlaylist big object;
+var firstDiv = document.createElement('div');
```

Comment:
Don;t know that this is but it's huge. Split it down in functions by responsability of each section